bin/load_exp_design.sh

```diff
@@ -16,16 +16,21 @@ sdrf_file=${SDRF_FILE:-$3}
 
 # Reason for creating this array is to search factor value column
 # In some sdrf files this column is mentioned as "Factor Value" and in some as "FactorValue"
-FactorArray=( FactorValue "Factor Value" )
-
+FactorArray=( FactorValue "Factor Value" "Factor Value " "FactorValue ")
+CharacteristicsArray=( Characteristics "Characteristics " )
```

Comment:
I’m not the designated reviewer, but I think [pattern matching](https://www.linuxjournal.com/content/pattern-matching-bash) is a more robust solution.